src/main/java/com/google/api/generator/engine/ast/StringObjectValue.java

```diff
@@ -14,6 +14,7 @@
 
 package com.google.api.generator.engine.ast;
 
+import com.google.api.generator.engine.escaper.StringValueEscaper;
```

Comment:
It looks like this is used only in StringObjectValue. What is the motivation for putting it in a separate package?